test/Client/Grpc.Tests/Client.Grpc.Tests.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net48</TargetFrameworks>
```

Comment:
remove net48 as I have a build error in VS  with this framework, saying that this csproj refers testhelper.csproj which only uses net6. I think we should be fine to remove this as we no longer support net48 anymore.

Reply:
Please add it back and rebase on `main`, I have addressed the issue